openhtf/io/output/upload_json.py

```diff
@@ -0,0 +1,72 @@
+"""Module for uploading JSON-formatted test record files to server."""
+
+import os
+import logging
+import requests
+
+from json import JSONEncoder
+from openhtf import conf
+from openhtf.io.output.json_factory import OutputToJSON
+
+conf.Declare('project', 'project name')
```

Comment:
Agreed, no need to complexify the configuration unnecessarily (this is our fault for setting a bad example, we've done this a lot in the past), let's roll this into a single string URL.